src/man/sssd-ipa.5.xml

```diff
@@ -536,6 +536,18 @@
                             from trusted domains directly while on a client
                             it will ask an IPA server.
                         </para>
+                        <para>
+                            NOTE: There are currently some assumptions that
```

Comment:
those must be fulfilled...

Reply:
@amitkumar50 what did you mean by this comment?

We would like to push this PR

Reply:
@lslebodn, @amitkumar50's suggestion is to change "There are currently some assumptions that must be met when SSSD is running on an IPA server."   to "There are currently some assumptions those must be fulfilled when SSSD is running on an IPA server".

@amitkumar50, is that your suggestion? @justin-stephenson, do you agree with the proposed change?

Reply:
Sorry for delay.

I suggested to use this line instead:
There are currently some assumptions `those must be fulfilled` when SSSD is running on an IPA server.
instead of
There are currently some assumptions `that must be met` when SSSD is running on an IPA server.

Since `assumptions` is Plural. and `that` is singular.